src/mangoapp.cpp

```diff
@@ -19,12 +19,15 @@ struct mangoapp_msg_v1 {
 
     uint32_t pid;
     uint64_t visible_frametime_ns;
-    uint8_t fsrUpscale;
-    uint8_t fsrSharpness;
+    uint8_t fsrUpscale;     // deprecated
```

Comment:
This actually showed whether FSR was doing anything not whether it was selected.

it 1280x800 game on 1280x800 output would show disabled here even if FSR was disabled.

Reply:
Okay, I understand. With a quick look at the sources it seems to be only activated if upscaling is necessary.

https://github.com/ValveSoftware/gamescope/blob/7a1fe2def38e9c3ed4d12adb9f1bb81b6f9dcbe5/src/steamcompmgr.cpp#L2263-L2265

_Off-topic: While I'm not knowledgeable in this matter, it might also be beneficial to filter / sharpen images that aren't (up-)scaled. #702_

-----------------------

To be technology agnostic: The same behavior applies to NIS.

There isn't any feedback currently, as the `g_bFSRActive` flag and `GAMESCOPE_FSR_FEEDBACK` atom explicitly reference FSR and only get triggered if FSR is used. I know nothing about Xlib and how this atom is used, but we should consider refactoring these technology specific bits. I can't find any other tool / program online, that references this specific atom though.

-----------------------

Nevertheless, if there is a reason to only use the filters on upscaled frames and keep the flag, it might be better to add a `scaler_filter_active` and keep `fsrUpscale` deprecated.

BTW: `fsrUpscale` still gets populated at the moment, so we are able to check / show this in MangoApp.

Reply:
It would be okay to repurprose this and the atom for a generic 'upscaling active'

Reply:
Regarding MangoApp, repurposing / renaming is denied (https://github.com/flightlessmango/MangoHud/pull/1042#discussion_r1221722402), adding fields would be the way to go for the IPC.

I can refactor the internal variables / logic though, so that it is generic.
You should decide if it is okay to rename the atom.